webdriver/tests/bidi/emulation/set_geolocation_override/contexts.py

```diff
@@ -103,3 +103,4 @@ async def test_multiple_contexts(
     assert (
         await get_current_geolocation(bidi_session, new_context) == default_coordinates
     )
+#Test
```

Comment:
```suggestion
#Test

```

Reply:
As the commit message states, it's added to trigger the tests. I'll remove it before merging.

Reply:
I do not see the comment but nevertheless you will take care.